Tools/parametric_model/generate_parametric_model.py

```diff
@@ -102,14 +100,61 @@ def start_model_estimation(config, log_path, data_selection="none", plot=False,
             visual_dataframe_selector_config_dict["sub_plt3_data"].append("fisher_information_rot")
 
         model.load_dataframes(select_visual_data(model.data_df,visual_dataframe_selector_config_dict))
+        print("Interactive data selection completed.")
+
+    # Setpoint based data selection
+    elif data_selection=="setpoint":
+        print("Setpoint based data selection enabled...")
+
+        selector = data_handler.config.selection_variable
+
+        if (selector not in ['aux1', 'aux2', 'aux3', 'aux4', 'aux5', 'aux6']):
```

Comment:
We don't want to necessarily limit the selector to `aux` variables. Could we just make it to check whether it is empty(`None`)?

Reply:
yes agree - you could eg also use the rudder stick for it (and disable manual yaw inputs)